samples/net/sockets/echo/prj_cc3220sf_launchxl.conf

```diff
@@ -0,0 +1,22 @@
+# Networking Config:
```

Comment:
We are trying to migrate away from having a new "full" conf file for each board. Please see for example how samples/net/echo_server is doing it and place board specific options into boards/ directory.

Reply:
Thanks for the review.

Yes will move conf file into /boards.